app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -59,7 +59,7 @@
     <string name="msg_this_room_is_read_only">Este chat é apenas de leitura</string>
     <string name="msg_invalid_2fa_code">Código 2FA inválido</string>
     <string name="msg_invalid_file">Arquivo inválido</string>
-    <string name="msg_invalid_server_url">URL de servidor inválida</string>
+    <string name="msg_invalid_server_url">URL de servidor inválido</string>
```

Comment:
I think this was correct before. Wouldn't it be "**A** URL está inválida"?

Reply:
Well, bear with me as I'm portuguese and it makes more sense to me that way. :smile:

Reply:
Oh, do say "O URL" instead there?

Reply:
Yes, we do. That's why it makes more sense to me that way. But feel free to use your gramatic and semantic  there. :smile: